### PR TITLE
Welcome message is shown to the user in Dashboard, when changing the bottom navigation framgents

### LINKED ISSUE #805: Welcome message is shown to the user in Dashboard,  when changing the bottom navigation framgents.
Welcome message should be only shown to the user after succesfull login.

### PR DESCRIPTION
**Pull request by cdhiraj40**
fixes #805

### CODE REVIEW

**Comment by cdhiraj40** [comment]
@asaikarthikeya I have made the PR, you can checkout.

File: org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java [hunk 1]
```diff
@@ -216,7 +216,7 @@
     private List<SizeSyncTextView> indicatorSyncGroup;
     private AppUpdateManager appUpdateManager;
     private Task<AppUpdateInfo> appUpdateInfoTask;
-
+    public static int count=0;
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java:219]: No need to define it as `public static`. Why not just use a boolean flag?

File: org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java [hunk 2]
```diff
@@ -248,9 +249,15 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
 
-        //
+
         this.updateUserLogin(userHandler.getUser());
 
+        // userHandler.getUser()!=null --> to check if user is logged out, userHandler.getUser() will give null if user logged out
+        if(count==0 && (userHandler.getUser()!=null))
+        this.ShowMessageOnce(userHandler.getUser());
+        //count will let message show only once whenever app/activity is launched.
+        count++;
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java:259]: Please, follow common Java code conventions and indent the lines it properly.

File: org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java [hunk 3]
```diff
@@ -661,7 +668,14 @@ public void onUserStatisticsUpdateEvent(final UserStatisticsUpdateEvent event) {
             userDurationTextView.setText(formatTimeForDashboard(event.totalDuration));
         });
     }
+    //show message when the app/this activity  is launched
+private void ShowMessageOnce(User user){
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java:672]: Violation of Java naming conventions. Methods start with lower case.

File: org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java [hunk 4]
```diff
@@ -203,19 +207,37 @@ protected void onStart() {
...
+                            @Override
+                            //finish the activity
+                            public void run() {
+                                finish();
+                            }
+                        };
+                        //post delay after login, 1000ms --> 1sec
+                        // so for around 1 sec welcome message will be shown
+                        handler.postDelayed(x, 1000);
+
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java:234]: You could rather start an intent for the `BaseMainActivity`, which would be much more straightforward.

**Review by SebaDro — COMMENTED** [review]
Many thanks to your proposed implementation. Let me first mention some formal aspects.
Please, take into account common Java code and naming conventions for future requests. By violating these formal aspects, PRs will be closed without any feedback. In addition, I would suggest you to find more straighforward solutions. 

Now, let's talk about your implementation. Actually, it provides a solution for the described issue. However, it's more like a workaround, which does not tackle the underlying problem which I have described in detail in #807.

If you wish to continue working on the underlying problem, please create a new PR that relates to it.

In the meantime, a more lightweight workaround has been provided by dba654c2 and b71ba2cb. Hence, I will close this PR.

**Comment by cdhiraj40** [comment]
@SebaDro Hey sorry about all the formal aspects which I didn't pay attention to. From next time I will make sure that I don't forget those aspects. 
Also sorry to say but I would have liked to change the code and make the requested changes as you asked earlier but it's okay I won't make mistakes again.

**Comment by SebaDro** [comment]
Yes, I totally agree without. Unfortunately, due to project related constraints it has to be fixed in a timely manner. Hope, you don't mind. There are a lot of open issues that are waiting for your contribution ;-)

**Pull request closed without merging**